lightning/src/util/events.rs

```diff
@@ -632,13 +632,13 @@ pub enum Event {
 		/// The `user_channel_id` value passed in to [`ChannelManager::create_channel`] for outbound
 		/// channels, or to [`ChannelManager::accept_inbound_channel`] for inbound channels if
 		/// [`UserConfig::manually_accept_inbound_channels`] config flag is set to true. Otherwise
-		/// `user_channel_id` will be 0 for an inbound channel.
+		/// `user_channel_id` will be randomized for an inbound channel.
```

Comment:
Not a big deal, but could say that the version it starts being randomized in

Reply:
Will include it in a follow-up, probably when having a look at #1800!

Reply:
Addressed in https://github.com/lightningdevkit/rust-lightning/pull/1855.